### PR TITLE
Add ArgochainTestnet configuration

### PR DESCRIPTION
**Pull request by pavondunbar**
feat: add Argochain Testnet (https://github.com/hyperlane-xyz/hyperlane-registry/pull/395)
### Description

 Add Argochain Testnet using the projects' official
docs.

### Backward compatibility

 No

### Testing

 CLI

---------

Authored-by: Pavon Dunbar <pavon@devolvedai.com>

### CODE REVIEW

File: chains/argochaintestnet/logo.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+	</g>
+</g>
+</svg>
```
Comment (paulbalaji) [inline chains/argochaintestnet/logo.svg:10]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" style="enable-background:new 0 0 1080 1080" viewBox="28.8 30.4 1019.1 1019.2"><path d="M28.8 540A509.8 509.8 0 0 1 544.9 30.4 509.5 509.5 0 1 1 28.8 540zm300.5 175.6a199.9 199.9 0 0 0-44.4 56.2c-8.1 16-13 32.7-13.2 50.7-.1 35 14.4 64.2 37 89.7 24.7 28 56.2 45.9 90.9 58.6a401.2 401.2 0 0 0 150 23c42.4-1 83.8-7.5 124-20.8 39.8-13 76.6-31.4 106-61.9 31.6-32.6 45-71.8 37.5-116.8-7.3-44-31.3-78-68.2-102.7a238 238 0 0 0-69.3-30.4l.7 1.5 5.3 3.8c21 15 40.8 31.2 56.2 52.1 29.2 39.8 24.2 78.9-13.9 110.2A198 198 0 0 1 671 860a406.8 406.8 0 0 1-122.2 24.2A382 382 0 0 1 426.5 873c-30.6-8.4-59.2-20.8-81.6-44.2a94.4 94.4 0 0 1-26.5-84.7c1.7-10 7.3-19.5 10.9-28.5zm54-358-5.7-5c-20.6-17.6-40.4-36-54.3-59.7a68.7 68.7 0 0 1-11.3-40.6 87.8 87.8 0 0 1 26.3-55c21-21.6 47.2-34 75.3-43A410.4 410.4 0 0 1 548 137.3c39.8 1 78.9 5.9 117 17.1 29.2 8.6 57 20.2 80.4 40.1 32.4 27.6 39.1 63.3 19 100.7-14.2 26.4-35.4 46.7-58.4 65.2-9.6 7.7-19.5 15-30.6 23.4 29.4-8.2 56.2-18.4 79.5-35.8C793.8 319 817 280.9 819 232c1.8-45.8-17.7-82.6-52.3-111.8a276 276 0 0 0-100-51.8 443.1 443.1 0 0 0-181.3-15.7c-40.8 4.8-80 14.9-116.7 33.6-35 18-64 42.2-82.5 77.7-21.9 42-22.1 83 6.2 122.2 23 31.8 54.5 53 88.5 71.2.4.2 1 0 2.6 0zm385.5 332.2-1.4 1.7 4.7 4.2a162.3 162.3 0 0 1 35.7 42.5c34.4 58.8 28 127.4-16.8 176.5a238.5 238.5 0 0 1-87.1 58.5 433.4 433.4 0 0 1-177.4 31.2c-42-1-83.1-7-122.9-20.7-41.6-14.3-79-35.3-107.5-69.8a144 144 0 0 1-34.4-77.8 127.8 127.8 0 0 1 22.8-85c20-30.4 46.8-53.7 76.2-74.2 8.6-6 17.4-11.5 27-17.8-1.7.1-2.3 0-2.8.3-46.8 18.7-89.3 43.9-123.5 81.5a230.9 230.9 0 0 0-62 145.6c-1 19.3-.8 19.4 13.7 31.8l1.8 1.6A492.4 492.4 0 0 0 521 1038.7c85.6 3.2 166.8-15 243.3-53.8A501 501 0 0 0 907.1 876c2.7-3 4-5.6 3.3-9.9-2.8-18-8.5-35-16.4-51.3-20.2-41.5-51-73.9-88.2-100.6-12-8.6-24.6-16.3-37-24.5zm-365-23.8c-12.8 5.7-44.7 31.7-58 47.9a69.2 69.2 0 0 0-17 31.7 84.2 84.2 0 0 0 15.6 66c16 21.7 38.2 34.6 62.7 43.9 44.8 16.9 91.5 20.7 138.8 18.4 45-2.1 89-10.1 130.9-27.7 22.7-9.6 44-21.5 59.7-41.3a55.6 55.6 0 0 0 8.3-59.6c-5.5-13.2-14-24.4-23.7-34.7-12-12.8-25.6-24-40.4-33.6 1 3.4 2.7 6.4 4.1 9.5 13.8 29.4 7.5 57-19 75.8-11.2 7.9-23.9 14.6-36.9 18.9-55.8 18.7-112.7 19-169.8 6.4-22.5-5-44-13-61.6-28.7-18.2-16.2-23.6-37.3-13.6-59.5 5.5-12.2 13.7-23.1 19.8-33.4zm303-321.1c24-20.3 46.2-42 55.8-73a56 56 0 0 0-9.9-53.8 129 129 0 0 0-23.6-22.7 207.4 207.4 0 0 0-69.7-31.6 459.4 459.4 0 0 0-170-13.9 303.8 303.8 0 0 0-100.8 25.6c-22.8 10.8-43.4 24.6-56.2 47.2-8.3 14.7-12.8 30.1-7.8 47.3 6.3 21.8 33.5 53.3 52.4 67.1l-1.5-3.5c-10.8-23.4-7.2-43.7 11.3-61.6a116.8 116.8 0 0 1 45-26 384 384 0 0 1 209.6-9.6c20.2 4.5 39.4 12 55.9 25 22.8 17.8 28.7 41.3 17.3 68-2.3 5.2-5.1 10.3-7.7 15.5zM421 370.3c2-16.5 6.8-24.6 20.8-32.8a127 127 0 0 1 24.4-11.3c34-11.3 69.4-14 105-13 24.7.8 49.4 3 72.7 11.8 17.4 6.5 33.8 14.5 34.6 37.1 11.2-10.2 23-27.5 27.8-40.3a42.2 42.2 0 0 0-13.2-50.2 108.2 108.2 0 0 0-40.4-21c-66.4-19.2-133-17.6-199.5-.3a133.3 133.3 0 0 0-58 28.6c-16.4 14.9-19.4 32.9-8.8 52.4a136.2 136.2 0 0 0 34.6 39zm210.4 55.5c4 0 6.6.3 9 0 24.1-3 48.3-5.1 72.2-9.4 39.2-7 76.7-19.4 110.3-41.3 58-37.4 87.8-90.4 87-160-.1-6.1-1.8-11.2-6.6-15.5-5.7-5-10.7-10.8-16.1-16.2C841 138.1 788 103.3 728.3 78.6c-7-2.9-14-5.5-21-8.2 1.4 2 3.3 3 5.2 3.8 28 12.9 53.8 29 75.3 51.4A141 141 0 0 1 828 248.3C821 296 795.8 333 756.3 360c-31.3 21.3-66.7 32.3-103.3 40.2-2 .4-4.7.9-5.7 2.3-5.4 7.4-10.4 15-16 23.3zm290.1-205.4c-.6.9-.8 1-.8 1.2 0 2-.2 3.9-.3 5.8a185.9 185.9 0 0 1-72 142.7c-32 25.8-69 41-108.5 51.3-35 9-70.7 13.6-106.8 15.4-2.5.1-6.5.8-7.2 2.4-4 9.4-7.4 19-11 28.8 1.8.6 2.4 1 2.9.9l11.7-.3c46.1-1 91.9-5.4 137.1-14.4 47.5-9.5 93.3-23.8 135.4-48.4a217.8 217.8 0 0 0 75.5-69.2c7-11 10.2-20.9 1-31.8l-.3-.5a500.4 500.4 0 0 0-56.7-83.9zM653.1 659c-.6 14-6.3 24-16.7 30.3-9.4 5.7-19.5 11-30 14.3a256 256 0 0 1-114 6.7c-17.3-2.5-34.2-7-48.8-17.4a32.5 32.5 0 0 1-14.6-20.6c-.7-4-.1-8.4-.1-14-16.1 15-30.2 29.5-36.9 49.5a37.4 37.4 0 0 0 9.5 40 101 101 0 0 0 45.6 25.5 309.4 309.4 0 0 0 154 5 144.6 144.6 0 0 0 58.2-24c14-10.5 23.4-23.9 21.6-42.4-2-20.7-14-36.4-27.8-52.9zm339.3-326.4c-4.8 7.3-8.5 13.8-12.9 19.8-23.6 32.2-55.1 54.8-90.4 72.5-61.3 30.7-127.2 44.3-194.9 50.5-25.7 2.4-51.6 2.9-77.4 4.4-1.6.1-4.4 1.6-4.6 2.8-1.6 8.1-2.6 16.4-3.8 25h34.4c73-.9 145.6-5.2 217.8-16.8 38.9-6.2 77.3-14.6 114-29.3 16.8-6.8 33-14.6 46.4-27.1 3.1-2.9 4.2-5.7 2.9-9.9-6.6-20.8-12.8-41.9-19.6-62.6-3.3-9.8-7.8-19.2-11.9-29.3zm-541 95.1.9-1.5c-4.2-6.6-8.1-13.3-12.7-19.5-1.4-2-4.5-3.7-7-4-32.8-3.4-65.2-9-96.3-20.2-37-13.3-69.8-33-93.8-65-27.8-37-36.4-79.5-33-124.9.6-10 2.3-20 3.6-30.4l-2.2 1.5a493.1 493.1 0 0 0-87 99c-1.6 2.2-2 6-1.4 8.6 5.3 25.8 19.6 46.3 38.2 64.1 25.3 24.3 55.7 40.4 88 53a541.8 541.8 0 0 0 122.2 30.4c26.7 3.7 53.6 6 80.5 9zm24.2 80.6-4.7-32.5c-10 0-19.2.2-28.4 0C369 473.4 296 467 224 451.8c-42-8.9-83.3-20.4-121.7-40.2-13.5-7-26-15.6-40.3-24.3-5.1 22.8-10.4 45.7-15.3 68.8-.3 1.6 1.8 4.8 3.5 5.5 10 4.5 20 9.2 30.3 12.5 44.6 14.5 90.7 21 137.1 25.7 74.7 7.7 149.7 8.8 224.8 8.3h33.3zm-362.8-229-1 2a474.3 474.3 0 0 0-43.3 91c-.5 1.7.5 4.8 1.8 5.9 6.6 5.2 13.1 10.6 20.3 15 34 20.7 71.3 32.9 109.6 42.4 79.3 19.7 160.1 26.9 241.5 29.6 8.5.2 17 0 26.4 0-3-7.9-5.9-15.5-8.5-23.2-1-3-2.8-4.1-6-4.3-37.1-2.1-74.1-6-110.8-12.7-46.2-8.5-91.3-20.6-133.1-42.5-43.9-23-80.6-53-96.9-103.2zm924 251.6c-11.5 2.8-22.8 6-34.3 8.1-53.8 9.6-108.2 11.8-162.6 14-26.2 1-52.5 1.4-78.7 1.6-40.2.4-80.4.3-120.5.4H611c2.1 9 4.2 16.8 5.9 24.8.8 3.5 2.4 4.6 6 4.7 33.1.9 66.2 1.4 99.3 3 52 2.7 104 5.3 156 9.7 35.4 3 70.7 8.5 105.9 14 15.5 2.4 30.5 7.7 45.8 11.6 4.2-11.7 8.6-69.6 7-92zM44.3 472.9c-4.4 18.6-6.6 80.6-3.6 95.6 22.5-3.1 44.8-7.5 67.4-9 63-4.3 126.2-8 189.4-10.7 48.5-2 97-2.3 145.6-3.3 10.2-.2 20.3 0 31.1 0l1.6-26.7h-36.2c-77.4.5-154.7-.9-231.7-9.3-36.4-4-72.7-9-108-18.7-18.9-5.1-37.2-11.9-55.6-17.9zM461 598.4c-18.7 0-36.3-.2-54 0-63.3 1.2-126.4 5.2-189 14.8-40.6 6.2-80.7 14.2-118.9 29.6-13 5.2-25.1 12.6-37.6 19-4.9 2.4-6.5 5.7-4.5 11.3 4 11 7 22.5 10.9 33.5 4 11.4 8.6 22.5 13.2 34.4 4.6-5.8 8.2-10.8 12.4-15.4 18.9-20.7 42.7-34.2 68-45.2 62.1-27 128-38.5 194.8-45.9 29.1-3.3 58.5-4.8 87.8-7.3 1.8-.1 4.3-1.3 5-2.7 4.1-8.4 7.8-17 11.9-26zm11.4-42c-16.2 0-31.5-.2-46.7.1-47.4 1-94.8 2.1-142.2 3.5-64.8 2-129.6 4.8-194 12-16.3 1.8-32.7 3.3-49.3 10.3l11.5 72.8c6-3.7 10.4-6.6 15-9.1 26.4-14.7 55-23.5 84.1-30.5 73.5-17.8 148.4-23.8 223.6-26.8 28.4-1 56.8-1 85.2-1.4 3.4 0 5.4-.6 6.3-4.3 2-9 4.4-17.8 6.5-26.5zM622 596.2c3.5 8.3 7.3 16.6 10.7 25.1 1.4 3.6 3.5 4.7 7.4 4.8 55.9 1.5 111.5 5.2 166.8 13 41.6 6 82.7 14 122.2 28.3 23.7 8.6 46.5 18.8 65.4 36.1 3.8 3.5 7.2 7.5 11.1 11.6 9.3-24.5 16-48.9 21.5-73.7 1-4.7-.2-7.2-4.5-8.5-13.3-3.8-26.4-8.6-40-11.2-69.8-13.4-140.6-17.5-211.4-21.3-34-1.8-68-3-102.1-4-15.4-.6-30.9-.2-47.1-.2zM1028 443c-4.2 3.3-7 5.6-10 7.8a236.6 236.6 0 0 1-67.7 30.6c-62.1 19-126.2 27.4-190.7 31.6-48.9 3.1-97.8 3.8-146.7 5.6-1.3 0-2.6.3-4.6.5.3 7 .8 13.6 1 20.2 0 4 1.7 5.2 5.7 5.2 12.3-.1 24.7.4 37 .3 56.2-.6 112.5-1 168.7-2 56-1.1 112-3.8 167.4-12.1 14.4-2.2 28.5-6.7 42.7-10.2 4.6-1.1 6.5-3.7 5.8-9-1.5-11.2-2-22.6-3.3-33.9-1.3-11.2-3.3-22.3-5.2-34.6zm-600.8-51 .1-1.9c-9.7-4-19.5-7.8-29-12.2-38.2-17.3-73.6-38.5-102.3-69.6-39.8-43.2-47-94-20.7-146.5 18.6-37.1 48.1-63.2 84.2-82.5 11-6 22.7-10.9 34.3-16.3-.8 0-1.4-.2-1.9 0A494.5 494.5 0 0 0 231.1 147a15.2 15.2 0 0 0-4.7 7.7 208 208 0 0 0-6 74.5A154.5 154.5 0 0 0 284.7 344a243.2 243.2 0 0 0 94 39.5c16 3.5 32.3 5.7 48.4 8.4zm-43 250.8-2-.2-11.8 1.1a868.2 868.2 0 0 0-138.6 23.9c-36 9.6-71.1 21.8-102.6 42.3-15.9 10.4-30.4 22.3-39.4 39.5-1.7 3.2-2.1 5.9-.2 9.3 10.2 17.6 20.1 35.3 30.2 53 1.7 3 3.8 5.7 6 8.9 11.6-34.2 35.9-62 61.4-83.1 39.2-32.6 84.2-54.8 131.6-72.5 21.3-8 43.2-14.7 65.4-22.2zm322.4 5.3-.3 1.2 6.9 2.9c53.4 21.8 102.8 49.9 143.3 91.7a234.2 234.2 0 0 1 61 104.3l2.9 11.6 1.7.7c12.7-18 25.6-36 38-54.2 1.9-2.7 2.8-7.3 1.9-10.3-3.2-9.8-6.2-20-11.4-28.7-14.3-24-35.8-40.8-59.3-55-39.7-24.1-83-39-127.7-50.8-18.9-4.9-38-9-57-13.4zM188.1 896c4.7-103.3 57.8-173.8 143.1-223.7-1.6.1-3 .4-4.4 1-23.3 10.2-47 19.5-69.7 30.9a292 292 0 0 0-91.2 69 132.4 132.4 0 0 0-30.4 54.3c-.7 2.7 0 6.7 1.7 9 7.1 9.6 14.5 19 22.3 28 9 10.5 18.5 20.5 28.6 31.5zm253.7-507.3c10.2-4.7 19.2-10.2 29-13.1 44.9-13.5 90.7-13.8 136.6-6.4 15 2.5 29.6 6.2 42 19 5.4-7.4 11.2-14.2 16-21.7 4-6.4 2.2-13-2.8-18.3a26.7 26.7 0 0 0-8.8-6.8c-10.6-4.3-21.3-9.2-32.4-11.6-41-8.8-82.1-8-123.2-.8-17.2 3-34.2 7.4-49.4 16.4-19.7 11.7-22 25.9-7 43.3.2.3.6.5 0 0zm10 251.5c-2.9 4.5-7.2 9.9-10.2 15.9-5 10-3.5 19 5.6 25.6 7.4 5.3 15.7 10 24.3 12.5 41 12.1 82.3 11.2 123.6 1.5a86.5 86.5 0 0 0 38.1-17.4c9.5-8.2 11.4-17.8 5.6-29-2.1-4-5-7.7-6.7-10.3-60.1 22.2-119 21.8-180.2 1.2zm521 146.9c8.5-17.8 16.9-35.7 25.5-53.4 1.9-4 2-7.4-.9-10.3-7.3-7.5-14.2-15.8-22.7-21.8-25.5-17.9-54.9-27.4-84.7-35.3a843.5 843.5 0 0 0-131.8-22.5c-9.3-1-18.6-1.6-27.9-2.4 16 4.8 32.3 8.6 48.3 13 41.7 11.7 82.4 26.4 119.5 49.4 32.5 20.2 60.7 44.5 74.8 83.3zM461.5 423c54-15.8 106.9-19.7 160 1.3l18.8-29.7c-5.5-7.2-13.7-9.7-21.7-12-25.2-7-51-8-77-7.2-24.5.8-48.9 3.1-72 11.9-6.5 2.4-12.5 6.4-18.3 10.3-1.3.9-2.2 4.7-1.4 6.2 3.6 6.7 7.8 13 11.6 19.2zm150.2 172.5c-46.6 19.8-93.4 17.9-140.6 3.2L458 629.9c5.8 2.4 11.3 5.1 17 6.7 40.1 10.8 80.5 10.7 121 2.8 10.4-2 20.5-5 29.3-11l-13.6-32.8zm-6.9-126.8 11.9-34.7c-50.2-19.7-100-16-150.1-1.4l11.9 33.3c42.2-10.5 84.4-11.8 126.3 2.8zM474.5 588.6c45.3 14 89.7 16 133.8-3.4a35551 35551 0 0 0-7-30.9c-39.8 9.7-79.3 12-118.8.7l-8 33.6zm128-109.3c-40.2-14.5-80.8-13.3-121.8-2.9l4.2 30.9h114l3.6-28zm-118.8 65c39.2 11.3 77.7 9.2 116.2-.7l-1.2-25.4H485.3l-1.6 26.1z" style="fill:#000"/></svg>
```

File: chains/argochaintestnet/metadata.yaml
Comment (paulbalaji) [inline chains/argochaintestnet/metadata.yaml:1]: please also add block information such as https://github.com/hyperlane-xyz/hyperlane-registry/blob/aa6e8bf36dbe75ca8b7f4a1ba784e61ae5fd4e58/chains/abstracttestnet/metadata.yaml#L7-L10

Comment (pavondunbar) [reply chains/argochaintestnet/metadata.yaml:1]: Block information has been added to metadata.yaml on lines 6-9 for your review.  Thank you.

File: chains/argochaintestnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - name: Argochain Testnet Explorer
+    url: https://test-scanner.devolvedai.com
+    apiUrl: https://test-scanner.devolvedai.com/api
```
Comment (paulbalaji) [inline chains/argochaintestnet/metadata.yaml:5]:
```suggestion
  - apiUrl: https://test-scanner.devolvedai.com/api
    family: other
    name: Argochain Testnet Explorer
    url: https://test-scanner.devolvedai.com
```

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**